modules/ROOT/pages/monitoring/concepts.adoc

```diff
@@ -0,0 +1,46 @@
+= Monitoring concepts
+:description: This section describes the concepts that are important to understand how Neo4j Ops Manager can be used for monitoring systems.
+
+This section describes the concepts that are important to understand how Neo4j Ops Manager can be used for monitoring systems in an optimal way.
+
+== Conditions
+
+A **Condition** specifies the constraints when the monitored entity reaches a certain state.
```

Comment:
Again with the caps. Unless you are changing the commonly accepted usage of the word _condition_, I'd recommend to not capitalize.